pkg/controller/volume/persistentvolume/delete_test.go

```diff
@@ -162,6 +162,28 @@ func TestDeleteSync(t *testing.T) {
 				return testSyncVolume(ctrl, reactor, test)
 			},
 		},
+		{
+			// PV requires external deleter
+			name:            "8-10-2 - external deleter when volume is provisioned by user",
```

Comment:
I am not clear about this, what does it mean by provisioned by user? could you tell me the difference between this one and previous test?

Reply:
It refers to static volume provisioning. User create both pv and pvc. the pv was not annotated like dynamic volume provisioning.

Reply:
you can see **What this PR does / why we need it:** in pr description for more details.

Reply:
Change it to static provisioning